src/main/webapp/app/entities/result/updating-result.component.ts

```diff
@@ -61,25 +64,12 @@ export class UpdatingResultComponent implements OnInit, OnChanges, OnDestroy {
         } else {
             const exercise = this.participation.exercise;
 
-            if (this.participation.results && this.participation.results.length > 0) {
-                if (exercise && exercise.type === ExerciseType.MODELING) {
```

Comment:
I think it's not a good idea to check for the exercise type here, we should always find the latest result from the results array.